community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/IndexingService.java

```diff
@@ -545,28 +547,29 @@ private void applyRecoveredUpdates() throws IOException, IndexEntryConflictExcep
                     updater.remove( recoveredNodeIds );
```

Comment:
I don't understand why we need to do this. Seems like when we are in `IndexUpdateMode.RECOVERY` we will use `SwallowingIndexUpdater` which is just no-ops.